content/ios/user.md

```diff
@@ -141,3 +141,25 @@ See [Friends and Followers]({{< relref "relation.md" >}}).
 ## Logging in with provider
 
 Right now to allow social plugin such as Facebook, plugin code must be written to enable it on the backend. Read more about in the [Plugin]({{< relref "plugin.md" >}}) section.
+
+# User Profile
+
+Whenever a new user signs up, a user profile is automatically created for
+you to track user information other than their _username_, _email_ or _password_.
+You can access the user profile the same way as accessing a record:
```

Comment:
I believe the JS guide of this paragraph has been modified after you write this. So you can re-copy this from the current master